src/Logary.Facade/Facade.fs

```diff
@@ -405,7 +403,7 @@ module internal Formatting =
           // but using 'LevelFatal' so it really stands out.
           themedParts.Add (prop.ToString(), LevelFatal)
 
-      FsMtParser.parseParts context.Buffer template foundText foundProp
```

Comment:
Dumping the string builder results in better performance, and it wasn't thread safe either.